src/content/app/species/components/species-sidebar-modal/modal-views/SpeciesSidebarDownloads.tsx

```diff
@@ -26,53 +26,69 @@ import ExternalLink from 'src/shared/components/external-link/ExternalLink';
 import styles from './SpeciesSidebarDownloads.module.css';
 
 const SpeciesSidebarDownloads = () => {
+  //Fetch an array of ftp links and convert to object
   const activeGenomeId = useAppSelector(getActiveGenomeId) || '';
   const { data: fileLinksResponse } = useSpeciesFileLinksQuery(activeGenomeId, {
     skip: !activeGenomeId
   });
   const fileLinks = fileLinksResponse?.links || [];
-  const linkSections: Record<string, string> = {};
-
+  const fileLinkURLs: Record<string, string> = {};
   for (const link of fileLinks) {
-    linkSections[link.dataset_type] = link.path;
+    fileLinkURLs[link.dataset_type] = link.url;
   }
-
+  //Annotations from Andy
+  type LinkAnnotation = {
+    type: string;
+    title: string;
+    text: string;
+  };
+  type LinkSection = {
+    title: string;
+    links: LinkAnnotation[];
+  };
+  const linkAnnotations: LinkSection[] = [
+    {
+      title: 'Assembly',
+      links: [{ type: 'assembly', title: 'DNA sequence', text: 'FASTA' }]
+    },
+    {
+      title: 'Annotation',
+      links: [
+        {
+          type: 'genebuild',
+          title: 'Genes, cDNA, ncRNA, proteins',
+          text: 'FASTA/GTF/GFF3'
+        },
+        { type: 'variation', title: 'Variation', text: 'VCF' },
+        { type: 'homologies', title: 'Homologies', text: 'TSV' },
+        {
+          type: 'regulation',
+          title: 'Regulation',
+          text: 'BigBED/TSV/GFF3/BigWig'
+        }
+      ]
+    }
+  ];
+  //Build the content
```

Comment:
Unnecessary comment (stating the obvious)

Reply:
Reverted